### PR TITLE
feat: fetch prices from Aquarius in AssetList component

### PR DESCRIPTION
**Pull request by LucaMilanese90**
## Proposed Changes

  - fetch prices from Aquarius instead of Provider for the asset list view

### CODE REVIEW

**Comment by moritzkirstein** [comment]
We have lost a few features / designs in asset lists:
grafik

- `Dataset` / `Algorithm` chip
- Name of publisher should be loaded from credential
- Network below price

File: src/@utils/aquarius/index.ts
Comment (nklomp) [inline src/@utils/aquarius/index.ts:305]: This is removing an additional filter present in FMDM

File: src/@utils/aquarius/index.ts
Comment (nklomp) [inline src/@utils/aquarius/index.ts:295]: This is a custom filter in FMDM

File: src/@utils/aquarius/index.ts
Comment (LucaMilanese90) [reply src/@utils/aquarius/index.ts:305]: Sorry for that, I'll push a fix

**Review by moritzkirstein — APPROVED** [review]

**Comment by moritzkirstein** [comment]
Waiting for approval from @nklomp or Sphereon team before merging

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**
